spicy-analyzer/analyzer/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+spicy_add_analyzer(
+    NAME @name@
+    PACKAGE_NAME @name@
+    SOURCES analyzer.spicy analyzer.evt zeek_analyzer.spicy
```

Comment:
It would be great to use concrete names here where the file name matches the module name.